demonstrations/learning2learn.py

```diff
@@ -0,0 +1,1205 @@
+r"""
+Learning to Learn with Quantum NN via Classical NN
+==================================================
+*Author: Stefano Mangini (mangini.stfn@gmail.com)*
+
+.. meta::
+    :property="og:description": Use a classical recurrent neural network to initilize the parameters of a variational quatum algorithm.
+    :property="og:image": ../demonstrations/learning2learn/thumbnail.png
+
+.. related::
+
+   tutorial_qaoa_intro QAOA
+   tutorial_qaoa_maxcut QAOA for MaxCut problem
+
+
+In this tutorial we recreate the architecture proposed by Verdon et
+al. in the paper *Learning to learn with quantum neural networks via
+classical neural networks* [#l2l]_, using **PennyLane** and **TensorFlow**.
+In this paper, classical recurrent neural networks are used to assist
+the optimization of variational quantum algorithms.
+
+We will start with a brief theoretical overview explaining the problem
+and the setup used to solve it. After that, we will deep dive into the
+code to build a fully functioning model, ready to be further developed
+or customized for your own needs. Without further ado, let’s start!
```

Comment:
```suggestion
or customized for your own needs. Without further ado, let’s begin!
```
To avoid repeated use of "start"